### ISSUE TITLE
[Bug]: Auto-forwarding does not work in send flow

### ISSUE THREAD

**Issue by aldertnl** [synonymdev/bitkit-react-native#746]
### Describe the bug

Bitkit does not forward the user to the Review & Send view in the Send flow after scanning a QR or pasting and invoice or address using the paste button. Instead, the user currently has to manually tap [Continue]. 

### Reproduce

1. Go to [Send]
2. Tap on [Scan] or [Paste] icon
3. Scanned invoice or copied invoice is pasted in the input field 
4. User has to manually tap [Continue] to go to Review&Send step.

Expected behavior: App forwards user automatically to Review&Send after step 2.

### Operating system

iPhone 11, iOS 16.1.1

### Bitkit version

v1.0 (23)

**Comment by pwltr** [comment]
Duplicate of https://github.com/synonymdev/bitkit/issues/441
